release-please-config.json

```diff
@@ -1,8 +1,7 @@
 {
   "packages": {
-    ".": {
-      "releaseType": "node",
-      "include-component-in-tag": false
```

Comment:
does it change the tag names for github releases?

Reply:
Yes, tags are now by package

Reply:
Will there still be a single release entry on GitHub? or multiple (per package)? can we change that?

Reply:
There will be multiple release entries on GitHub. We can't change this (and we shouldn't because the releases are necessary for our token).